### ISSUE TITLE
Multiple LHS vars don't work

### ISSUE THREAD

**Issue by grantmcdermott** [grantmcdermott/ggiplot#1]
```r
library(ggiplot)
library(ggplot2)
library(fixest)

mods = feols(c(mpg, wt) ~ i(vs, drat), mtcars)

iplot(mods)                          # works
ggiplot(mods)                        # "works" (bad title)
ggiplot(mods, multi_style = 'facet') # doesn't


## Non-facettng case doesn't work at all in the presence of other multi-model
## options, e.g. csw(). 
mods2 = feols(c(mpg, wt) ~  csw(disp, qsec) + i(vs, drat), mtcars)
iplot(mods2)                          # works
ggiplot(mods2)                        # doesn't
ggiplot(mods2, multi_style = 'facet') # doesn't
```

**Comment by grantmcdermott** [comment]
Above examples all work (with some improvements over the `iplot` equivalents IMO). Note that ggplot2 is also loaded automatically with ggiplot and no longer needs to be called separately.

``` r
library(ggiplot)
#> Loading required package: ggplot2
library(fixest)

mods = feols(c(mpg, wt) ~ i(vs, drat), mtcars)

ggiplot(mods)                        
#> Warning: Removed 1 rows containing missing values (geom_vline).
```

![](https://i.imgur.com/SbOYaaq.png)

``` r
ggiplot(mods, multi_style = 'facet') 
#> Warning: Removed 2 rows containing missing values (geom_vline).
```

![](https://i.imgur.com/3u2oSqK.png)

``` r
## Multi-model on both lhs and rhs
mods2 = feols(c(mpg, wt) ~  csw(disp, qsec) + i(vs, drat), mtcars)
ggiplot(mods2)                        
#> Warning: Removed 1 rows containing missing values (geom_vline).
```

![](https://i.imgur.com/sDPOCfz.png)

``` r
ggiplot(mods2, multi_style = 'facet')
#> Warning: Removed 4 rows containing missing values (geom_vline).
```

![](https://i.imgur.com/vqqYmJ1.png)

<sup>Created on 2022-03-31 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

**Comment by grantmcdermott** [comment]
Broken again in latest dev version.

**Comment by grantmcdermott** [comment]
Fixed by #19 (which adds some overdue tests too).
